.readthedocs.yml

```diff
@@ -0,0 +1,7 @@
+version: 2
+
+python:
+  version: "3.8"  # To ensure that we can install the sphinx-2i2c-theme
+  install:
+    - requirements: requirements.txt
+ 
```

Comment:
```suggestion
# Configuration on how ReadTheDocs (RTD) builds our documentation
# ref: https://readthedocs.org/projects/2i2c-team-compass/
# ref: https://docs.readthedocs.io/en/stable/config-file/v2.html
version: 2

build:
  os: ubuntu-20.04
  tools:
    python: "3.8"

sphinx:
  configuration: conf.py

# Explicitly opt out of trying to build additional formats such as PDF and ePub
formats: []

python:
  install:
    - requirements: requirements.txt
```

If you want to ever update to Python 3.9, you must use the non-deprecated way of installing Python, which is like this.

I added some misc stuff I think can be helpful to have declared in a RTD file as well by copy-pasting from other configs.